e2e/integration/utils/getToken.js

```diff
@@ -1,18 +1,21 @@
 const jwt = require('jsonwebtoken');
 const fs = require('fs');
-const {promisify} = require('util');
+const { promisify } = require('util');
 const jwtSign = promisify(jwt.sign);
 const readFile = promisify(fs.readFile);
 
 const jwtOptions = {
   algorithm: 'RS256',
   issuer: 'tweek',
-  expiresIn: '15m'
+  expiresIn: '15m',
+  keyid: 'test-tweek-keyid',
```

Comment:
Remote test-tweek-keyid, not needed.
Separate flows of openid-connect and Tweek issuer

Reply:
Added issue #757